### PR TITLE
Add dev.krtirtho.Flemozi

### PR DESCRIPTION
**Pull request by KRTirtho**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://docs.flathub.org/docs/for-app-authors/requirements
[maint]: https://docs.flathub.org/docs/for-app-authors/maintanance
[submission]: https://docs.flathub.org/docs/for-app-authors/submission

### CODE REVIEW

**Comment by KRTirtho** [comment]
bot, build dev.krtirtho.flemozi

**Comment by KRTirtho** [comment]
bot, build dev.krtirtho.Flemozi

**Comment by KRTirtho** [comment]
bot, build dev.krtirtho.Flemozi

**Comment by KRTirtho** [comment]
bot, build dev.krtirtho.Flemozi

File: flahub.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+  "only-arches": [
+    "x86_64"
```
Comment (hfiguiere) [inline flahub.json:3]: this is disappointing

File: dev.krtirtho.Flemozi.yml [hunk 2]
```diff
@@ -0,0 +1,31 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/flatpak/flatpak-builder/main/data/flatpak-manifest.schema.json
+
+app-id: dev.krtirtho.Flemozi
+runtime: org.freedesktop.Platform
+runtime-version: "22.08"
```
Comment (hfiguiere) [inline dev.krtirtho.Flemozi.yml:5]: 23.08 is out.

File: dev.krtirtho.Flemozi.yml [hunk 3]
```diff
@@ -0,0 +1,31 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/flatpak/flatpak-builder/main/data/flatpak-manifest.schema.json
+
+app-id: dev.krtirtho.Flemozi
```
Comment (hfiguiere) [inline dev.krtirtho.Flemozi.yml:3]: while `krtirho.dev` answer, it has an invalid certificate.

File: flahub.json (hunk 1 above)
Comment (KRTirtho) [reply flahub.json:3]: I'm still searching ways to build flutter for arm64 arch :sweat_smile:

File: dev.krtirtho.Flemozi.yml (hunk 3 above)
Comment (KRTirtho) [reply dev.krtirtho.Flemozi.yml:3]: You had a typo missing a "t"  https://krtirtho.dev. Although I guess this is not the reason

I own the domain but I haven't yet published any site so never needed to certify the primary domain (Not experienced enough to manually certify without help of any service :sweat_smile: )
Btw, there's couple of subdomains I use that are certified .e.g [spotube.krtirtho.dev](https://spotube.krtirtho.dev/) & [fl-query.krtirtho.dev](https://fl-query.krtirtho.dev/)

**Comment by KRTirtho** [comment]
bot, build dev.krtirtho.Flemozi

File: dev.krtirtho.Flemozi.yml (hunk 3 above)
Comment (hfiguiere) [reply dev.krtirtho.Flemozi.yml:3]: the verification relies on the domain to work.

Comment (KRTirtho) [reply dev.krtirtho.Flemozi.yml:3]: Check now. I've updated certificates & stuff

Comment (KRTirtho) [reply dev.krtirtho.Flemozi.yml:3]: Also there's a present for you
image

**Pull request closed without merging**